File: 0222-count-complete-tree-nodes/0222-count-complete-tree-nodes.java
/**
 * Definition for a binary tree node.
 * public class TreeNode {
 *     int val;
 *     TreeNode left;
 *     TreeNode right;
 *     TreeNode() {}
 *     TreeNode(int val) { this.val = val; }
 *     TreeNode(int val, TreeNode left, TreeNode right) {
 *         this.val = val;
 *         this.left = left;
 *         this.right = right;
 *     }
 * }
 */
class Solution {
    public int countNodes(TreeNode root) {
        LinkedList<TreeNode> ll = new LinkedList<>();
        ll.add(root);
        int c = 0;
        while(!ll.isEmpty()) {
            TreeNode curr = ll.pollFirst();
            if(curr==null)
                continue;
            c++;
            ll.add(curr.left);
            ll.add(curr.right);
        }
        return c;
    }
}

File: 0222-count-complete-tree-nodes/0222-count-complete-tree-nodes.cpp
/**
 * Definition for a binary tree node.
 * struct TreeNode {
 *     int val;
 *     TreeNode *left;
 *     TreeNode *right;
 *     TreeNode() : val(0), left(nullptr), right(nullptr) {}
 *     TreeNode(int x) : val(x), left(nullptr), right(nullptr) {}
 *     TreeNode(int x, TreeNode *left, TreeNode *right) : val(x), left(left), right(right) {}
 * };
 */
class Solution {
public:
    int countNodes(TreeNode* root) {
        queue<TreeNode*> ll;
        ll.push(root);
        int c = 0;
        while(!ll.empty()) {
            TreeNode* curr = ll.front();
            ll.pop();
            if(curr==NULL)
                continue;
            c++;
            ll.push(curr->left);
            ll.push(curr->right);
        }
        return c;
    }
};